qa/L0_lifecycle/lifecycle_test.py

```diff
@@ -140,24 +140,23 @@ def test_parse_error_modelfail(self):
 
         # And other models should be loaded successfully
         try:
-            for base_name in ["savedmodel", 'netdef']:
```

Comment:
For all these lifecycle tests it is important the we don't just remove netdef but that we replace it with something else. An onnx model would be best.